lib/SIL/IR/OperandOwnership.cpp

```diff
@@ -278,39 +280,58 @@ OPERAND_OWNERSHIP(EndBorrow, EndBorrow)
 
 #undef OPERAND_OWNERSHIP
 
-// Conditionally either ForwardingConsume or ForwardingBorrow, depending on
-// instruction result's dynamic ownership kind.
-#define FORWARD_ANY_OWNERSHIP(INST)                                            \
-  OperandOwnership                                                             \
-  OperandOwnershipClassifier::visit##INST##Inst(INST##Inst *i) {               \
-    return i->getOwnershipKind().getForwardingOperandOwnership();              \
+// Forwarding operations are conditionally either ForwardingConsumes or
+// ForwardingBorrows, depending on the instruction result's dynamic ownership.
```

Comment:
Again, dynamic makes me nervous here. I would prefer if we never use the word dynamic since it makes it sounds like it is implicit when it is static, albeit changeable.

Reply:
An instruction's dynamic behavior includes anything that can change across instances of that instruction. That's a term of art as I understand it.

Reply:
I'm referring to it everywhere as "the instruction's constant ownership attribute."